spline-shadow/rest-core/src/main/scala/za/co/absa/spline/gateway/rest/model/ExecutedLogicalPlan.scala

```diff
@@ -22,24 +22,31 @@ import za.co.absa.spline.gateway.rest.model.ExecutedLogicalPlan._
 
 case class ExecutedLogicalPlan
 (
-  app: AppInfo,
   execution: ExecutionInfo,
-  dag: LogicalPlan
-)
+  plan: LogicalPlan
+) {
+  def this() = this(null, null)
+}
 
 object ExecutedLogicalPlan {
   type OperationID = UUID
 
-  case class LogicalPlan(nodes: Seq[Operation], edges: Seq[Transition]) extends Graph {
+  case class LogicalPlan(nodes: Array[Operation], edges: Array[Transition]) extends Graph {
```

Comment:
Is this generic enough name? I mean, I don't think logical plan has good equivalent in StreamSets.

Reply:
Maybe you're right. What about just ExecutionPlan? I think you can't really get more generic than that. The only doubt that I have is that if we happen to also store a physical plan (and we forget about YAGNI)  then the work "logical" here would help to avoid confusions.

Reply:
exec plan sounds good or executionLineage.